src/main/kotlin/org/rust/ide/docs/DocumentationUtils.kt

```diff
@@ -0,0 +1,177 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.docs
+
+import com.intellij.codeInsight.documentation.DocumentationManagerUtil
+import com.intellij.psi.PsiElement
+import org.rust.ide.utils.escaped
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+
+internal fun PsiElement.generateDocumentation(buffer: StringBuilder, prefix: String = "", suffix: String = "") {
```

Comment:
`internal` doesn't make sense in our codebase, as it means *private outside module* not *private outside package* as Java's `package-local` does

Let's just make it public, eventually leave a comment that this function is package internal